open-metadata-implementation/access-services/digital-service/README.md

```diff
@@ -0,0 +1,25 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the ODPi Egeria project. -->
+
+![InDev](../../../open-metadata-publication/website/images/egeria-content-status-in-development.png#pagewidth)
+
+# Digital Service Open Metadata Access Service (OMAS)
+
+The Digital Service OMAS provides APIs and events for tools that play a role in the lifecycle of an egeria Digital Service. 
```

Comment:
Remove Egeria
Need to describe what a digital service - link to definition in gdata-govenrance.git.